packages/cozy-client/src/CozyClient.js

```diff
@@ -483,7 +483,8 @@ class CozyClient {
     }
     const withIncluded = await this.fetchRelationships(
       mainResponse,
-      this.getIncludesRelationships(definition)
+      this.getIncludesRelationships(definition),
+      definition.cursor
```

Comment:
I am a bit confused, maybe I've missed something but it seems here that we are using the parent definition (the one to fetch albums) to store the `cursor` for a specific relationship. What happens if we have two relationships from the same query ? Shouldn't the `cursor` be specific to a single relationship ?